protocol-units/bridge/chains/movement/src/lib.rs

```diff
@@ -0,0 +1,211 @@
+use aptos_sdk::{
+	move_types::language_storage::TypeTag,
+	rest_client::{Client, FaucetClient},
+	types::LocalAccount,
+};
+use aptos_types::account_address::AccountAddress;
+use bridge_shared::{
+	bridge_contracts::{
+		BridgeContractCounterparty, BridgeContractCounterpartyError,
+		BridgeContractCounterpartyResult,
+	},
+	types::{
+		Amount, BridgeTransferDetails, BridgeTransferId, HashLock, HashLockPreImage,
+		InitiatorAddress, RecipientAddress, TimeLock,
+	},
+};
+use rand::prelude::*;
+use serde::Serialize;
+use std::str::FromStr;
+use url::Url;
+
+use crate::utils::MovementAddress;
+
+pub mod utils;
+
+const DUMMY_ADDRESS: AccountAddress = AccountAddress::new([0; 32]);
+const COUNTERPARTY_MODULE_NAME: &str = "atomic_bridge_counterparty";
+
+enum Call {
+	Lock,
+	Complete,
+	Abort,
+	GetDetails,
+}
+
+pub struct MovementClient {
+	///Address of the counterparty moduke
+	counterparty_address: AccountAddress,
+	///Address of the initiator module
+	initiator_address: Vec<u8>,
+	///The Apotos Rest Client
+	rest_client: Client,
+	faucet_client: FaucetClient,
+	///The signer account
+	signer: LocalAccount,
+}
```

Comment:
I notice several project-specific configurations, such as the Suzuka configuration and dot movement settings, within our main framework that compromise its modularity by adding unnecessary complexity. 

To maintain the architectural integrity and clarity of our system, I propose that we either remove or refactor these configurations. Please let me know your thoughts on this matter.

Reply:
Yes fine, I was just doing what I needed to stand this up, again everything config can be done later and merged into the feature bracn at a later date.